core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -3763,7 +3765,7 @@ LogicalProject(DEPTNO=[$7])
     </TestCase>
     <TestCase name="testReduceConstExpr">
         <Resource name="sql">
-            <![CDATA[select sum(case when 'y' = 'n' then ename else 1 end) from emp]]>
+            <![CDATA[select sum(case when 'y' = 'n' then ename else 0.1 end) from emp]]>
         </Resource>
```

Comment:
Is this change correct?

Reply:
+1. The test case should be using `1`: https://github.com/apache/calcite/blob/37b8cdbb381369e773229a81ae69ab5c1df34f3e/core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java#L2649

Reply:
Got it, 3q.

Reply:
Yes, the 'sql' originates in `SqlToRelConverterTest.java`. I just updated `SqlToRelConverterTest.xml` to match what was already in the java.